lib/faraday/error.rb

```diff
@@ -38,6 +38,14 @@ def inspect
     #              :headers - String key/value hash of HTTP response header
     #                         values.
     #              :body    - Optional string HTTP response body.
+    #              :request - Hash
+    #                         :method   - Symbol with the request HTTP method.
```

Comment:
looking at line 36, seems like indentation is off and you need to add 2 whitespaces (I might be wrong)

Reply:
I think you're right, I'll change it!